XPathTest/Qt3Tests/Qt3TestRunners.cs

```diff
@@ -28,18 +28,18 @@ public Qt3TestRunners(TestLoggingFixture loggingFixture)
         RunTest(name, testSetName, description, testCase, arguments, nodeUtils);
     }
 
-    [Theory(Timeout = 60000, DisplayName = "Qt3 Tests: LINQ XML API")]
```

Comment:
I think you don't want to merge this,right?

Reply:
If you're doing this often, consider making it a flag or something to enable/disable linq tests